### DISCUSSION TITLE
convert this tool to window

### DISCUSSION THREAD

**Discussion by leitao1019** [chris1111/HP-Probook-EliteBook-Package-Creator#66] [Ideas]
dear Chris,
i have an old notebook HP Probook 6470b, run with win10 very well.
i'd like to install the MacOS into it for development purpose.
looks like you have done a lot of work for HP Probook & EliteBook with MacOS.
but i can not run it in windows, sorry that i do not have an MacOS currently.
is it possible to run your tools in windows or WSL?

thanks in advance~

**Comment by chris1111** [comment]
no its for macOS usage

**Reply by leitao1019** [reply]
OK, got it. thanks for your response~
